environment/environment.go

```diff
@@ -456,13 +466,32 @@ func (env *k8sEnvironment) doPortForward(pod *coreV1.Pod, forwarder *portforward
 	}
 }
 
+func (env *k8sEnvironment) deployChainlinkSecrets() error {
```

Comment:
Kludge, tired of dealing with the weird secrets bug. Coming back to this in the A.M.

We might want something similar to this though. It's a bit more efficient to have all the nodes just use the same secret, less to deal with in handling secret names (using `generateName` makes us need to retrieve the final one after deployment), which might be a headache. Plus it's more efficient. 

The only issue would be that, well, we need to use the same secrets values for all nodes. Not sure how much that matters though, especially at this point.